js/App/AddMemberEvent.js

```diff
@@ -0,0 +1,91 @@
+import {
+  $todoAppContainer,
+  $todoListItem,
+  $appMemberAdd,
+} from "../content/shape.js";
+import { addMember, responseMemberApi } from "../server/AppServer.js";
+import {
+  countContainer,
+  initFilterEventListeners,
+} from "./todoAppCountContainer.js";
+
+import {
+  clickEraseButton,
+  clickInput,
+  clickCheckboxButton,
+  clickLabel,
+  checkPriority,
+} from "./mainLiItem.js";
+
+//member add  관련
+function makeAddList(value) {
+  value.innerHTML = $appMemberAdd;
+}
+
+function addMemberEvent(teamId) {
+  const $addTeamButton = document.querySelector("#add-user-button");
+  $addTeamButton.addEventListener("click", () => addMemberEventHandler(teamId));
+}
+
+async function addMemberEventHandler(teamId) {
+  const result = prompt("팀 이름을 입력해주세요");
+  if (/[\S]/gi.test(result) && result !== null) {
+    addMember(result, teamId);
+  } else alert("공백 혹은 정상적이지 않은 팀 이름 입력");
+}
+
+//member별 item
+function loadMemberList(teamMember, button, todoList, teamId, memberId) {
```

Comment:
매개변수가 많을 때에, 호출을 할 때 사용자의 실수가 있을 수 있습니다. 

이럴 때 destructure 문법을 사용하여 호출할 때  매개변수이름을  명시적으로 사용 하면 좋습니다 
```suggestion
function loadMemberList({ teamMember, button, todoList, teamId, memberId } ) {
```